stone/target/python_rsrc/stone_validators.py

```diff
@@ -18,12 +18,13 @@
 import numbers
 import re
 import six
-from typing import Optional
+import typing  # noqa: F401 # pylint: disable=unused-import
 
+# See <http://python3porting.com/differences.html#buffer>
 if six.PY3:
-    _binary_types = (bytes, memoryview)
+    _binary_types = (bytes, memoryview)  # noqa: E501,F821 # pylint: disable=undefined-variable,useless-suppression
 else:
-    _binary_types = (bytes, buffer)
+    _binary_types = (bytes, buffer)  # noqa: E501,F821 # pylint: disable=undefined-variable,useless-suppression
```

Comment:
Depending on which version of Python is being used, one of `memoryview` and `buffer` doesn't exist. Hence, the weird mix of `undefined-variable,useless-suppression` on both.